lint.ignore

```diff
@@ -726,6 +726,12 @@ MISSING DEPENDENCY: web-nfc/resources/nfc-helpers.js
 MISSING DEPENDENCY: webusb/resources/usb-helpers.js
 MISSING DEPENDENCY: webxr/resources/webxr_util.js
 
+# TODO(Hexcles): delete these files once we include them in mojojs.zip.
```

Comment:
I don't support we have a bug tracking this on the Chromium side to link to? If not, not super important.